setup.py

```diff
@@ -8,11 +8,19 @@
 with open("README.md") as readme_file:
     readme = readme_file.read()
 
+format_libs = {
+    "base-imageio": "imageio[ffmpeg]~=2.9.0",
+    "lif": "readlif~=0.5.1",
```

Comment:
Shouldn't `czi` be in here?

Reply:
Waiting for `CziReader`. Since it will rely on `aicspylibczi==3.0.0` I just removed until @heeler gets to that release + PR

Reply:
Got it!